### ISSUE TITLE
Last update(1.0.7) broke the imports

### ISSUE THREAD

**Issue by Dtsiantaris** [ps-cristopher/vue3-autocounter#12]
[plugin:vite:import-analysis] Failed to resolve entry for package "vue3-autocounter". The package may have incorrect main/module/exports specified in its package.json

Hey just wanna let you know since updating to 1.0.7 i got this error. 

I am using Vue 3.2.25

Downgrading to 1.0.6 fixed my issue.

Let me know if you want more info for reproduction. 

Greate work btw!! ✌️✌️

**Comment by xsorifc28** [comment]
It seems like the build job is broken, there is no `dist` folder in the npm package. 

Only the source is available, but this doesn't seem intended as package.json still has the entry points as:
```json
"main": "dist/vue3-autocounter.ssr.js",
"browser": "dist/vue3-autocounter.esm.js",
"module": "dist/vue3-autocounter.esm.js",
"unpkg": "dist/vue3-autocounter.min.js",
  ```
  @ps-cristopher

**Comment by Josepdal** [comment]
If anyone is facing this error, edit the version:

`"vue3-autocounter": "1.0.6" // Remove ^ if set`

If it persists, remove package-lock.json and node_modules and install everything again.

Hope @ps-cristopher can check it soon for the new users downloading this repo.

**Comment by ntgraph** [comment]
Hi, 
Will this update fixed soon?  
This library so far (v1.0.6) is nice.  Keep up the good work.

**Comment by ps-cristopher** [comment]
Hi all!, thanks for reporting it, I'll check what could have happened with the last release.

**Comment by ps-cristopher** [comment]
Solved on the last release [1.0.8 ](https://github.com/ps-cristopher/vue3-autocounter/releases/tag/v1.0.8)
